### DISCUSSION TITLE
New useful feature : search in a folder

### DISCUSSION THREAD

**Discussion by rskhiri** [sumatrapdfreader/sumatrapdf#3540] [Ideas]
Love the speed of SumatraPDF but can't search for some expression in a folder without opening pdf files.
It's really useful if you have tons on pdfs like invoices...
This feature is implemented in Foxit Reader but Foxit is too slow and gets bigger in each version, more than 300 Mo now

**Comment by GitHubRulesOK** [comment]
search in folder is available in slower Acrobat, slower Foxit, slower Tracker, and "Slower SumatraPDF".
Folder searching is **slow** if not managed well via user choice of index locations.

![image](https://github.com/sumatrapdfreader/sumatrapdf/assets/12754913/76f0f2cd-2bb4-4c35-85c7-ee895e549227)

Fastest windows app for Index searches is probably Agent Ransack, uses same index technology as all of above, but Explorer is the core.
